dom.bs

```diff
@@ -499,6 +505,16 @@ list of tuples, each of which consists of an <b>item</b> (an {{EventTarget}} obj
 object). A tuple is formatted as (<b>item</b>, <b>target</b>, <b>relatedTarget</b>). A <a
 for=Event>path</a> is initially the empty list.</p>
 
+For web compatibility, the <a>current global object</a> has an
+<dfn attribute for=Window><code>event</code></dfn> attribute that always returns the <a>current event</a>
+during <a>dispatch</a>, and is otherwise undefined. The <dfn type=dfn id=concept-current-event>current event</dfn>
+is the <var>event</var> passed to <a>inner invoke</a>. If <var>event</var>'s {{Event/target}} <a for=tree>root</a>
+is a <a for=/>shadow root</a>, the <a>current event</a> is undefined.
```

Comment:
This will need to become an imperative definition where we manipulate the internal slot that the attribute reflects from the dispatch algorithm. Declarative definitions often lead to subtle edge cases becoming undefined over time.